sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/ViewUtils.java

```diff
@@ -58,9 +129,15 @@ final class ViewUtils {
           }
         }
       }
-    }
 
-    return target;
+      if (AndroidComposeViewUtils.isComposeView(view)) {
```

Comment:
How about we make some abstraction here like 
```
interface TouchTargetFinder {
  UiElement findTarget(...);
}
```
then we could have a list of those in SentryOptions, and register them upon initialization. And then we could iterate through the list (should be ordered) and use the first found target.

Then we could have a default implementation for traditional views in `-core`, and have a `compileOnly` dependecy on `-compose` package as you suggested, and then check at runtime if the compose implementation is present - if yes, we'd add it to the beginning of the list of touchTargetFinders (or whatever the name is). And then we could just iterate through the list in `SentryGestureListener`. How does that sound?